src/Nethermind/Nethermind.Optimism/OptimismWithdrawals.cs

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Consensus.Withdrawals;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Specs;
+using Nethermind.Logging;
+using Nethermind.State;
+
+namespace Nethermind.Optimism;
+
+/// <summary>
+/// https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/exec-engine.md#l2tol1messagepasser-storage-root-in-header
+/// </summary>
+public static class OptimismWithdrawals
```

Comment:
Since this class only has one "inner" class I think we can merge them and have a single `class OptimismWithdrawalsProcessor : IWithdrawalProcessor`.